packages/account/src/Components/api-token/api-token.tsx

```diff
@@ -232,7 +215,6 @@ const ApiToken = ({ footer_ref, is_app_settings, overlay_ref, setIsOverlayShown
                                                     <ApiTokenCard
                                                         name='read'
                                                         value={values.read}
-                                                        setFieldValue={setFieldValue}
```

Comment:
👍 
`value` can also be removed as it can be handled by the `Field` inside `ApiTokenCard`

Reply:
The value for `value` prop changes based on scope of `ApiTokenCard`. We will still need to pass the scope from eg `values.admin` from parent card.

Reply:
We don't need to pass the value, Field will handle it by itself based on the unique name prop

Reply:
ref: https://formik.org/docs/api/field#name